plugin/pkg/admission/exec/admission.go

```diff
@@ -100,24 +112,21 @@ func (d *DenyExec) Validate(a admission.Attributes) (err error) {
 	if path != "pods/exec" && path != "pods/attach" {
 		return nil
 	}
-	pod, err := d.client.Core().Pods(a.GetNamespace()).Get(a.GetName(), metav1.GetOptions{})
+	pod, err := d.client.CoreV1().Pods(a.GetNamespace()).Get(a.GetName(), metav1.GetOptions{})
 	if err != nil {
 		return admission.NewForbidden(a, err)
 	}
 
-	if pod.Spec.SecurityContext != nil {
```

Comment:
👍🏻looks really good. am slightly not sure about this line b/c it's possibly changing server's behavior. is it just for a prevention of NPE or any other reason?

Reply:
answering myself. removing this line is fine.